src/Microsoft.AspNetCore.Mvc.Api.Analyzers/AddResponseTypeAttributeCodeFixAction.cs

```diff
@@ -64,7 +67,16 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
                 documentEditor.RemoveNode(attributeSyntax);
             }
 
-            return documentEditor.GetChangedDocument();
+            var document = documentEditor.GetChangedDocument();
+
+            var root = await document.GetSyntaxRootAsync(cancellationToken);
+
+            if (root is CompilationUnitSyntax compilationUnit && addUsingDirective)
+            {
+                root = compilationUnit.AddUsings(SyntaxFactory.UsingDirective(SyntaxFactory.ParseName("Microsoft.AspNetCore.Http")));
```

Comment:
Does this result in a compile error if the using already exists? Could you add a test that covers this?

Reply:
Nice catch!

> The using directive for 'Microsoft.AspNetCore.Http' appeared previously in this namespace